frontend/src/api/auth.ts

```diff
@@ -0,0 +1,101 @@
+import {AuthData, TokenPair, UserData} from "../model/auth";
+import axios, {AxiosError, AxiosRequestHeaders} from "axios";
+
+const url = 'http://localhost:3001/users/';
```

Comment:
Co znaczy 3001 port? Jest to port django?

Reply:
tak, zmieniłem na 3001 bo kolidował z react dev serverem